app/renderer/js/main.js

```diff
@@ -282,6 +305,12 @@ class ServerManagerView {
 			this.activateTab(index);
 		});
 
+		ipcRenderer.on('reload-proxy', () => {
+			this.loadProxy().then(() => {
+				alert('Proxy settings saved!');
```

Comment:
Do we need to show this alert when the user turns `off` the proxy option? How about this -

``` javascript
ipcRenderer.send('forward-message', 'toggle-sidebar', newValue);
```

```javascript
ipcRenderer.on('reload-proxy', (event, newValue) => {
	this.loadProxy().then(() => {
		if (newValue !== false) {
			alert('Proxy settings saved!');
		}
	});
});